### ISSUE TITLE
ITMS-90338: Non-public API usage when submitting due to FFmpeg framework

### ISSUE THREAD

**Issue by bighitbiker3** [tanersener/react-native-ffmpeg#102]
**Description**
There's an invalid binary error when submitting to the App Store due to a non-public API being used? I feel as if it's a false positive but I don't know enough about iOS development to be sure.  I'm using the `https-lts` pod

This is message from Apple - ITMS-90338: Non-public API usage - The app references non-public symbols in App: _locale_charset.

**Logs**
```
grep -Rnis '_locale_charset' *
Binary file ios/Pods/mobile-ffmpeg-https/libiconv.framework/libiconv matches
Binary file ios/Pods/mobile-ffmpeg-https/libcharset.framework/libcharset matches
```
**Environment**
- RN 0.60.5
- rn-ffmpeg - latest

**Comment by tanersener** [comment]
I regularly test App-Store submissions and I haven't seen this error before.

I tried to submit the [default test-app](https://github.com/tanersener/react-native-ffmpeg-test/tree/master/default) which uses react-native 0.60.5 for `https-lts` package and it didn't received any errors.

Can you compare your project dependencies/setting with [default test-app](https://github.com/tanersener/react-native-ffmpeg-test/tree/master/default)?

**Comment by tanersener** [comment]
Maybe one of them is enabling an option which triggers Apple's complaints.

**Comment by tanersener** [comment]
@bighitbiker3 do you really need https access in your application? You can depend on `min-lts` package if you don't need it. It will resolve the issue for you.

**Comment by bighitbiker3** [comment]
I don't. I've moved to `min-lts` and it's working fine! Thanks a lot for looking into this

**Comment by tanersener** [comment]
Great 👍

**Comment by ju5td0m7m1nd** [comment]
I think we should reopen this issue
since I need to use http-lts and it will be rejected by Apple

Also I using codepush as well

**Comment by tanersener** [comment]
I follow this issue on my parent project tanersener/mobile-ffmpeg/issues/230. I can open this issue as well but I don't have all the answers about it.

**Comment by ju5td0m7m1nd** [comment]
In my case, after I removed **react-native-code-push** related pod dependency, it shows no issue when I used https-gql-lts.
The added dependency for react-native-code-push is

- CodePush (6.0.0):
    - Base64 (~> 1.1)
    - JWT (~> 3.0.0-beta.7)
    - React
    - SSZipArchive (~> 2.1)

Anyone has the idea about what trigger Apple's complaints?

**Comment by tanersener** [comment]
This issue should have been fixed in `v0.4.1`. Can you please test it?
